sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -10,27 +10,16 @@
 - Enable updating the CMK encryption key (workspace.encryption.keyVaultProperties.keyIdentifier) for a workspace.
 - Mark JobService class and services param to command() as experimental.
 - Added a replication_count value to the schema of SystemCreatedStorageAccount in Registry.
-- Support expression in if-else in Pipelines.
-- Support command schedule in Pipelines.
-- Support Output & port type Input in @group in Pipelines.
-- Support parallel-for pipeline with subgraph as body in Pipelines.
-- Additional includes support azure devops artifacts for Component.
 - Added support for Fairfax and MoonCake cloud for the registry discovery baseurl.
-- Added support for schedule in spark job.
-- Added support for foreach node in SDK.
 - Added support for variable args as pipeline input in DSL Pipeline.
-- Added support for dsl condition with one branch in Pipeline.
 - Added OS Patching Parameters to Compute Instance.
 
 ### Bugs Fixed
 - Update the upper bound dependencies version for tqdm, strictyaml, colorama and opencensus-ext-azure.
 - Added missing "properties" to batch deployment.
 - Retain the cases for the names of system job services (Tracking and Studio).
-- Fixed Sweep node not excluding optional input with value `None` in REST object. 
```

Comment:
This Sweep line should be add back?

Reply:
No, I notice that there is one line "Fixed sweep job optional input cannot be empty." describing the same thing, then I removed my line.